Library/Homebrew/cmd/commands.rb

```diff
@@ -8,6 +8,18 @@ module Homebrew
   module_function
 
   def commands
+    valid_options = ["--quiet", "--include-aliases"]
+    invalid_options_by_user = []
+    ARGV.options_only.each do |option|
+      invalid_options_by_user.push << option unless valid_options.include?(option)
+    end
+    unless invalid_options_by_user.empty?
+      odie <<-EOS.undent
+        Invalid option#{"s" if invalid_options_by_user.length>1} provided: #{invalid_options_by_user.join " "}
```

Comment:
Move the `#{"s" if invalid_options_by_user.length>1}` out of the heredoc (and below).